### ISSUE TITLE
Progress indicator expressed in task button

### ISSUE THREAD

**Issue by TurkeyMan** [jake-phy/WindowIconList#110]
Windows 7+ has a really nice feature where apps presenting a progress indicator is represented with a progress indicator behind the text on the task button. Is there such APIs available to diplicate this functionality? It's super handy; copying large files, downloads in browsers, music apps show song position, Visual Studio shows project compiling progress...
It would be super awesome if bash processes with a sense of progress reflected on the taskbar (make, apt-get, etc), but I'm sure there's no api available to hook this information?

**Comment by jake-phy** [comment]
Alright this is a great idea, I shall see if I can tap into that.  I think I would like to get this released first.

**Comment by TurkeyMan** [comment]
Sure, no problem! Just thought I'd note some features I've really missed since you seemed to appear from the shadows ;)

**Comment by jake-phy** [comment]
Yeah np keep them coming.  I do that about once a year it seems :)
